test/integration/noderesourcetopology_test.go

```diff
@@ -600,6 +534,131 @@ func TestTopologyMatchPlugin(t *testing.T) {
 	}
 }
 
+func TestTopologyMatchReservePlugin(t *testing.T) {
+	testCtx, _, ns := makeTestFixture(t)
+	ctx := testCtx.Ctx
+	cs := testCtx.ClientSet
+
+	cfg := makeSchedulerComponentConfig(t, ns)
+	cfg.Profiles[0].Plugins.Reserve.Enabled = append(cfg.Profiles[0].Plugins.Reserve.Enabled, schedapi.Plugin{Name: noderesourcetopology.Name})
+
+	testCtx = util.InitTestSchedulerWithOptions(
+		t,
+		testCtx,
+		true,
+		scheduler.WithProfiles(cfg.Profiles...),
+		scheduler.WithFrameworkOutOfTreeRegistry(fwkruntime.Registry{noderesourcetopology.Name: noderesourcetopology.New}),
+	)
+	t.Log("Init scheduler success")
+	defer testutil.CleanupTest(t, testCtx)
+
+	// Create a Node.
+	resList := map[v1.ResourceName]string{
+		v1.ResourceCPU:    "4",
+		v1.ResourceMemory: "100Gi",
+		v1.ResourcePods:   "32",
+	}
+	for _, nodeName := range []string{"fake-node-1", "fake-node-2"} {
+		newNode := st.MakeNode().Name(nodeName).Label("node", nodeName).Capacity(resList).Obj()
+		n, err := cs.CoreV1().Nodes().Create(ctx, newNode, metav1.CreateOptions{})
+		if err != nil {
+			t.Fatalf("Failed to create Node %q: %v", nodeName, err)
+		}
+
+		t.Logf(" Node %s created: %v", nodeName, n)
+	}
+
+	nodeList, err := cs.CoreV1().Nodes().List(ctx, metav1.ListOptions{})
```

Comment:
The err is not used anywhere, which failed the integration test.

Reply:
thanks, bad rebase. Fixed! Going to add the actual test ASAP and then lift the WIP.